modules/skybrowser/skybrowsermodule.cpp

```diff
@@ -0,0 +1,692 @@
+                "Returns a list of all the loaded AAS WorldWide Telescope images that "
+                "has been loaded."
+            }, 
+            {
+                "setHoverCircle",
+                &skybrowser::luascriptfunctions::setHoverCircle,
+                "string",
+                "Takes an identifier to a screen space renderable and adds it to the "
+                "module."
+            },
+            {
+                "moveCircleToHoverImage",
+                &skybrowser::luascriptfunctions::moveCircleToHoverImage,
+                "int",
+                "Moves the hover circle to the coordinate specified by the image index."
+            },
+            {
+                "disableHoverCircle",
+                &skybrowser::luascriptfunctions::disableHoverCircle,
+                "",
+                "Disables the hover circle, if there is one added to the sky browser "
+                "module. "
+            },
+            {
+                "loadImagesToWWT",
+                &skybrowser::luascriptfunctions::loadImagesToWWT,
+                "string",
+                "Takes an identifier to a sky browser or target and loads the WWT image "
+                "collection to that browser."
+            },
+            {
+                "selectImage",
+                &skybrowser::luascriptfunctions::selectImage,
+                "int",
+                "Takes an index to an image and selects that image in the currently "
+                "selected sky browser."
+            }, 
+            {
+                "removeSelectedImageInBrowser",
+                &skybrowser::luascriptfunctions::removeSelectedImageInBrowser,
+                "string, int",
+                "Takes an identifier to a sky browser or target and an index to an "
+                "image. Removes that image from that sky browser."
+            },
+            {
+                "adjustCamera",
+                & skybrowser::luascriptfunctions::adjustCamera,
+                "string",
+                "Takes an identifier to a sky browser or sky target. Rotates the camera "
+                "so that the target is placed in the center of the view."
+            },
+            {
+                "setSelectedBrowser",
+                & skybrowser::luascriptfunctions::setSelectedBrowser,
+                "string",
+                "Takes an identifier to a sky browser or target. Sets that sky browser " 
+                "currently selected."
+            },
+            {
+                "getTargetData",
+                &skybrowser::luascriptfunctions::getTargetData,
+                "",
+                "Returns a table of data regarding the current view and the sky browsers "
+                "and targets."
+            },
+            {
+                "lockTarget",
+                &skybrowser::luascriptfunctions::lockTarget,
+                "string",
+                "Takes an identifier to a sky browser or target. Locks the target " 
```

Comment:
Do the target and sky browser have the same identifier?

It's a little confusing that these function can take the identifier of either of them. Is it actually the identifier of the **pair** of a target and browser that it wants?